src/style.css

```diff
@@ -0,0 +1,132 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+
+.page_body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card_block {
+  width: 200px;
+  height: 406px;
```

Comment:
do not hardcode height
card should be stretched by its content

Reply:
not fixed